third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -2,76 +2,306 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_classes_with_only_static_members
-// ignore_for_file: camel_case_types
-// ignore_for_file: non_constant_identifier_names
-// ignore_for_file: prefer_const_declarations
-
 /// ansi_up is an library that parses text containing ANSI color escape
 /// codes.
-@JS()
-library ansi_up;
-
-import 'package:js/js.dart';
 
-@JS()
 class AnsiUp {
-  external AnsiUp();
+  String version, _buffer;
+  bool bold;
+  List<List<AnsiUpColor>> ansiColors;
+  List<AnsiUpColor> palette256;
+  AnsiUpColor fg;
+  AnsiUpColor bg;
+  RegExp _csiRegex;
 
-  external TextPacket get_next_packet();
+  // ignore: sort_constructors_first
+  AnsiUp() {
+    version = '4.0.3';
```

Comment:
nit: remove version. We don't care as the Dart `pub` package system lets us track versions.